Add checklist and responsibility document for Fireside chat (#2985)
alan-turing-institute/the-turing-way: book/website/community-handbook/fireside-chat/fireside-chat-roles.md

(ch-fireside-chat-roles)=

# Fireside Chat Roles and Responsibilities

(ch-fireside-chat-roles-chair)=
## Chair - Opens the Session

> One of *The Turing Way* team members will open the panel, welcoming the audience, introducing *The Turing Way* and the Fireside Chat, and setting the norm (CoC, Etherpad, chat, question process). They’ll thank the co-hosts and pass the ‘mic’ to them. 

Author: ```suggestion
One of *The Turing Way* team members will open the panel, welcoming the audience, introducing *The Turing Way* and the Fireside Chat, and setting the norm (Code of Conduct [CoC], Etherpad, chat, question process). They’ll thank the co-hosts and pass the ‘mic’ to them. 
```

Facilitators are invited based on their experience on the topic and they also respond to questions, adding to the discussion from their experiences.

* Once agreed upon, all speakers will be invited to join a private Slack channel to participate in asynchronous planning discussions.
* They will supply a short bio and image to be shared on the Eventbrite page.
* They will be invited to attend a call (30) minutes before the event to discuss the topic and help the co-hosts identify what specific area of the topic they will talk about.
* They will be coordinating with the co-host to finalise the specific question posed to them, and if they would like, they can suggest another question for the other panellists.
* They will have access to the internal planning documents where they can find all information they need, as well as have opportunities to give feedback by directly collaborating on the Google document.
* They should check what in addition to questions posed to them, questions they would like to respond to as a follow-up discussion.
* Join Zoom at least 5-10 minutes before the event to test their system (microphone, camera, internet, and access to document used for the session)

> * It is important that all speakers get the same ‘stage time’ -- so please take this space to share your thoughts if you are generally not the one speaking out, and if you generally tend to speak more, please intentionally create space for others to speak.

Reviewer: From here down the person changes from "they will" (third person) to "you" (second person) - not sure which one we prefer to use

* We suggest not engaging in the Zoom chat to avoid getting distracted. The co-hosts will keep track of questions being asked there and share on the Etherpad, as well as pose to the panel if time allows.
* If we can’t cover all the audience questions, we encourage speakers to take 5 minutes after the event to respond on Etherpad.
* We will upload the video on YouTube, so if there are any parts of the discussion speakers would rather not have shared there, they should let *The Turing Way* team know.
* If the speakers have any written scripts while preparing for this panel (such as for the opening and closing statements), we request them to share those with *The Turing Way* team to help with their video transcription review work.

(ch-fireside-chat-roles-team)=
### *The Turing Way* Team - Supports the Event

> * Invite the co-hosts at least 6 weeks in advance and set up an initial planning discussion with them. This planning discussion allows co-hosts to explore the topics they want to speak about and the possible set of speakers they would like to invite from their network.
> * Following this discussion, *The Turing Way* team will suggest a few titles for the event on Slack and invite discussion to finalise the speakers to be invited.
> * Once speakers and titles are finalised, they will make sure that there is someone (co-hosts or from *The Turing Way* Team) delegated to contact the speaker using the email draft.
> * They will help set up the Eventbrite, Etherpad, Zoom and Flyer for online sharing -- inviting feedback and edits from the co-hosts and speakers. If a speaker hasn’t shared their bio and image, they will follow up.
> * They will share the speaker release form via email before the event.
> * They will upload the video on YouTube:
>   * *The Turing Way* team will invite speakers to help review the uploaded transcription for correctness.
> * They will send a “thank you email” to all the attendees via Eventbrite sharing the link to the YouTube video.
> * They will thank the speakers, and make sure that the signed speaker’s release form has been obtained. 

Author: ```suggestion
* Invite the co-hosts at least 6 weeks in advance and set up an initial planning discussion with them. 
* The planning discussion is crucial for *The Turing Way* team and co-hosts to explore the topics and themes they want to speak about and the possible set of speakers they would like to invite from their network.
* Following this discussion, the co-hosts will propose a few titles for the event and nominate speakers (via Slack).
* Based on the previous contacts with the proposed speakers, one of the co-hosts or a *Turing Way* team member will email the speaker sharing initial details and proposed theme. 
*The Turing Way* Team they will make sure that there is someone delegated to contact the speaker using a template email draft. 
They will also follow up and set up neetings for discussions as needed.
* They will help set up the Eventbrite, Etherpad, Zoom and Flyer for online sharing -- inviting feedback and edits from the co-hosts and speakers. If a speaker hasn’t shared their bio and image, they will follow up.
* They will send calendar invites for the event with link for joining the call.
* On the day of event, a team member will open the call, welcome participants and monitor chat and Etherpad.
* They will share the speaker release form via email before the event.
* They will upload the video on YouTube:
  * *The Turing Way* team will invite speakers to help review the uploaded transcription for correctness.
* They will send a “thank you email” to all the attendees via Eventbrite sharing the link to the YouTube video.
* They will thank the speakers, and make sure that the signed speaker’s release form has been obtained. 
```

* They will upload the video on YouTube:
  * *The Turing Way* team will invite volunteers and/or speakers to help review the uploaded transcription for correctness.
* They will send a “thank you email” to all the attendees via Eventbrite sharing the link to the YouTube video.
* They will thank the speakers, and make sure that the signed speaker’s release form has been obtained. 

(ch-fireside-chat-roles-agenda)=
## Event Agenda

Fireside Chats follow this rough agenda:

* The co-hosts will introduce the topic and the panel. The invited co-host will also introduce their organisation/community as co-organisers.
* Each speaker will give a 2-minute pitch covering their interests, experiences, concerns and insights related to the event's topic.
* Session hosts will state the question and invite responses from 1-2 speakers (the questions can be agreed upon in advanced) - there might be follow-up question based on the response that we may not have discussed in advance
* Hosts will make sure that all speakers have had the chance to speak (by responding to different questions) and express their thoughts or share resources from their work.

> * If time permits, we will take questions from the audience and speakers will be invited to address them. We will however invite attendees to use Zoom chat and Etherpad notes to share their thoughts.

Author: ```suggestion
* If time permits, co-hosts will take questions from the audience and speakers will be invited to address them.
They will also invite attendees to use Zoom chat and Etherpad notes to share their thoughts.
```

* Decide on the title - this will allow us to put together an Eventbrite page. 
* Decide on the speakers - they are invited and details are added on the Eventbrite page and other promotional material.
* If they already know the suggested speaker, we will appreciate it if they can send the invitation and get confirmation using an [email draft](https://docs.google.com/document/d/12VoVexsXXBx25SnAU4mayPw5un-v936g9Ux61DLjGzo/edit).
* Host 30 mins calls with the speakers before the event to identify what they would like to speak about.
* Frame questions based on their discussions with the speakers, and invite speakers to help finalise the question (using Slack and the planning document).
* Considering the flow of the discussion, decide on the order of the questions that will be asked.
  * Each co-host should have equal opportunity to get the ‘stage time’
  * They should agree on which questions they will pose to guide the discussion.
  * They should pose 1 question to each speaker who takes a lead on responding. 1-2 follow-up responses can be invited from other speakers.
    * Make sure that the speakers know in what order they will be speaking, and what questions will be asked.

>     * Coordinate via the Slack channel, and share any notes about the format via the shared document.

Reviewer 2: What happens if the host panels don't use Slack? Should we asked them which communications tool they would like us to use for coordinations?
